[FLINK-19478][docs-zh][python]  Translate page "intro_to_table_api" into Chinese (#13537)
apache/flink: docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md

    )
""")

> # 4. 从 source 表中查询同时执行计算

Reviewer: ```suggestion
# 4. 查询 source 表，同时执行计算
```

> # 使用 Table API 查询语句创建一张表：

Reviewer: ```suggestion
# 通过 Table API 查询语句创建一张表：
```
使用 -> 通过？其他地方也看一下是否有必要改动。

Author: `使用`改为`通过`是会通顺些，我看下下其它地方，可以改动的都改动了。

8> +U(3,19)
{% endhighlight %}

实际上，结果展示了打印接收器所接收到的变更日志。
变更日志输出的格式为:
{% highlight text %}
{subtask id}> {消息类型}{值的字符串格式}
{% endhighlight %}
例如，"2> +I(4,11)" 表示这条消息来自第二个 subtask，同时 "+I" 表示这是一条插入的消息。"(4, 11)" 是这条消息的内容。
另外，"-U" 表示这是一条回撤的消息 (即更新前)，这意味着应该在 sink 中删除或回撤该消息。 
"+U" 表示这是一条更新的记录 (即更新后)，这意味着应该在 sink 中更新或插入该消息。

所以，我们从上面的变更日志中得到以下结果：

{% highlight text %}
(4, 11)
(2, 15) 
(3, 19)
{% endhighlight %}

### Table API 和 SQL 的混合使用

Table API 中的 `Table` 对象和 SQL 中的 tables 可以自由地相互转换。

下面例子展示了如何在 SQL 中使用 `Table` 对象：

{% highlight python %}

> # 创建一张 sink 表来发送数据

Reviewer: ```suggestion
# 创建一张 sink 表来接收结果数据
```

    )
""")

# 将 Table API 表转换成 SQL 的视图
table = table_env.from_elements([(1, 'Hi'), (2, 'Hello')], ['id', 'data'])
table_env.create_temporary_view('table_api_table', table)

> # 将数据写入 Table API 表

Reviewer: ```suggestion
# 将 Table API 表的数据写入结果表
```

    )
""")

> # 将 sql 表转化成 Table API 表

Reviewer: ```suggestion
# 将 sql 表转换成 Table API 表
```

table = table_env.from_path("sql_source")

# 或者从 sql 查询语句中创建表
table = table_env.sql_query("SELECT * FROM sql_source")

> # 将数据发送到该表中

Reviewer: ```suggestion
# 将表中的数据写出
```

{% endhighlight %}

结果为：

{% highlight text %}
   id   data
0   1     Hi
1   2  Hello
{% endhighlight %}

> <span class="label label-info">Note</span> "to_pandas" 将会触发表的物化同时将表的内容收集到客户端内存中，所以通过 <a href="{{ site.pythondocs_baseurl }}/api/python/pyflink.table.html#pyflink.table.Table.limit">Table.limit</a> 来限制收集数据的条数是一种很好的做法。

Reviewer: ```suggestion
<span class="label label-info">Note</span> "to_pandas" 将会触发表的物化，同时将表的内容收集到客户端内存中，所以通过 <a href="{{ site.pythondocs_baseurl }}/api/python/pyflink.table.html#pyflink.table.Table.limit">Table.limit</a> 来限制收集数据的条数是一种很好的做法。
```

{% endhighlight %}

> ### 将数据发送到多张的 Sink 表中

Reviewer: ```suggestion
### 将数据发送到多张 Sink 表中
```

    )
""")

# 创建 statement set
statement_set = table_env.create_statement_set()

> # 发送 "table" 对象到 "first_sink_table"

Reviewer: ```suggestion
# 将 "table" 的数据写入 "first_sink_table"
```

statement_set.add_insert("first_sink_table", table)

> # 通过一条 sql 插入查询语句将数据从 "simple_source" 发送到 "second_sink_table"

Reviewer: ```suggestion
# 通过一条 sql 插入语句将数据从 "simple_source" 发送到 "second_sink_table"
```

7> +I(2,Hello)
{% endhighlight %}

Explain 表
-----------------

> Table API 提供了一种机制来翻译 `Table` 的逻辑和优化查询计划。 

Reviewer: ```suggestion
Table API 提供了一种机制来解释 `Table` 的逻辑查询计划和优化后的查询计划。 
```

> 这个通过 `Table.explain()` 或者 `StatementSet.explain()` 方法来完成。`Table.explain()` 返回 `Table` 的执行计划。`StatementSet.explain()` 返回多个 sink 的执行计划。这些方法返回一个字符串来描述三个方面：

Reviewer: ```suggestion
这是通过 `Table.explain()` 或者 `StatementSet.explain()` 方法来完成的。`Table.explain()` 可以返回一个 `Table` 的执行计划。`StatementSet.explain()` 则可以返回含有多个 sink 的作业的执行计划。这些方法会返回一个字符串，字符串描述了以下三个方面的信息：
```
